http4s/client/src/main/scala/endpoints4s/http4s/client/Endpoints.scala

```diff
@@ -243,23 +243,40 @@ trait EndpointsWithCustomErrors
       responseA(sc, hs)
         .map(_.andThen(_.map(_.asLeft[B])))
         .orElse(responseB(sc, hs).map(_.andThen(_.map(_.asRight[A]))))
-
-  //#endpoint-type
-  type Endpoint[A, B] = Kleisli[Effect, A, B]
+   //#endpoint-type
+  trait Endpoint[A, B] {
+    /** This method returns a Resource[Effect,B] unlike sendAndConsume this is always safe to use in regard to lazyness
```

Comment:
```suggestion
    /** This method returns a Resource[Effect, B] unlike `sendAndConsume` this is always safe to use in regard to laziness
```

Reply:
Could you maybe add a comment indicating _when_ the request is sent to the server? (Maybe this is obvious to the users of `Resource`, though?)

Reply:
It is possible that this depends on the actual chosen http4s client. so it's hard to say in general